### PR TITLE
added get high watermark fields and current execution timestamp to co…

### PR DESCRIPTION
**Pull request by AmandeepSinghCS**
added get high watermark fields

### CODE REVIEW

File: clouddq/lib.py [hunk 1]
```diff
@@ -195,6 +202,26 @@ def prepare_configs_from_rule_binding_id(
         {"configs_hashsum": sha256_digest(json.dumps(resolved_rule_binding_configs))}
     )
     configs.update({"progress_watermark": progress_watermark})
+    print("Incremental time fileter column")
```
Comment (nauh) [inline clouddq/lib.py:205]: Remove these "print" lines? (I assume it's for debugging)

**Review by nauh — COMMENTED** [review]
Do we have a test case for this? (in addition to the test which checks the expected template)

File: clouddq/lib.py (hunk 1 above)
Comment (AmandeepSinghCS) [reply clouddq/lib.py:205]: yes, modified it to logger.

**Comment by nauh** [comment]
> checks

**Pull request merged**